samples/subsys/fs/format/README.md

```diff
@@ -0,0 +1,33 @@
+# Format Sample Application
+
+## Overview
+This sample is designed to show how to format different storage devices for different
```

Comment:
"The aim of this sample application is to..."

Reply:
Updated.